docs/pages/router/advance/shared-routes.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Shared routes
+description: Learn how to define shared routes or use arrays to use the same route multiple times with different layouts using Expo Router.
+hideFromSearch: true
+---
+
+import { FileTree } from '~/ui/components/FileTree';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+You can prevent a segment from showing in the URL by using the group syntax `()`.
+
+- **app/root/home.js** which matches `/root/home`
+- **app/(root)/home.js** which matches `/home`
+
+This is useful for adding layouts without adding additional segments to the URL.
```

Comment:
```suggestion
You can prevent a segment from showing in the URL using the group syntax `()`. This is useful for adding layouts without adding additional segments to the URL.

- **app/root/home.js** which matches `/root/home`
- **app/(root)/home.js** which matches `/home`
```

Let's define why the usage of the group syntax is useful in the first line instead of breaking the flow. Often times, people are skimming through docs and easy to miss after the bullet points.